app/src/main/java/ai/elimu/content_provider/ui/image/ImagesFragment.kt

```diff
@@ -148,8 +148,10 @@ class ImagesFragment : Fragment() {
                 }
 
                 // Store the Image in the database
-                imageDao.insert(image)
-                Log.i(TAG, "Stored Image in database with ID " + image.id)
+                image?.let {
```

Comment:
@tuancoltech Is this necessary if `image` never will be null?

Reply:
@jo-elimu I've reviewed and found that it's safe to remove nullable question mark here, and in other places as well.
So, removed the nullability.